curl_request.hpp

```diff
@@ -292,6 +314,14 @@ class Request
         response.curlCode = curlCode;
         return response;
     }
+    
+    static size_t file_write_callback(void* ptr, size_t size, size_t nmemb, void* userdata)
```

Comment:
Put this function at the top with the other one, and format it in the same manner (`_m_fileWriteFunction`)